src/monsters.c

```diff
@@ -181,16 +188,18 @@ static order *get_money_for_dragon(region * r, unit * u, int wanted)
 
     /* falls der drache launisch ist, oder das regionssilber knapp, greift er alle an */
     n = 0;
-    for (u2 = r->units; u2; u2 = u2->next) {
-        if (inside_building(u2) != u->building && is_guard(u, GUARD_TAX) && u2->faction != u->faction && cansee(u->faction, r, u2, 0)) {
-            int m = get_money(u2);
-            if (m == 0 || is_guard(u2, GUARD_TAX) || attack_chance <= 0.0)
```

Comment:
Deswegen frage ich doch. Ich habe das jetzt so verstanden, dass das is_guard in 186 da war, damit Einheiten nicht doppelt angegriffen werden.

Noch was: War das so gedacht, das Drachen grundsätzlich PLÜNDERn sollen, oder sollen sie TREIBEn, falls das möglich ist?